packages/core/src/payment/strategies/bnz/bnz-payment-strategy.ts

```diff
@@ -0,0 +1,53 @@
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { HostedFormFactory } from '../../../hosted-form';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import PaymentActionCreator from '../../payment-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import { CardinalThreeDSecureFlowV2 } from '../cardinal';
+import { CreditCardPaymentStrategy } from '../credit-card';
+
+export default class BNZPaymentStrategy extends CreditCardPaymentStrategy {
```

Comment:
Only curiosity, why the name is `BNZ` instead of `BankOfNewZealand`, because in the future, BNZ could be difficult to understand to what provider is referring 💅🏻

Reply:
We decided to name it BNZ to avoid typos